client/app/assets/less/redash/redash-newstyle.less

```diff
@@ -804,6 +804,48 @@ text.slicetext {
     text-align: center !important;
   }
 
+  dashboard-page {
```

Comment:
In the future, please place the markup for the component/page in a file next to the component/page.